sanitise/base.go

```diff
@@ -1,5 +1,9 @@
 package sanitise
 
+import "github.com/electivetechnology/utility-library-go/logger"
+
+var log logger.Logging
+
```

Comment:
I can't find initialisation for the logger in the constructor. Is the logging working?